openapi-cli/src/test/resources/generators/client/ballerina/display_and_deprecated.bal

```diff
@@ -0,0 +1,92 @@
+        return response;
+    }
+    # Create a pet
+    #
+    # + return - Null response
+    remote isolated function createPet() returns http:Response|error {
+        string  path = string `/pets`;
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        http:Response response = check self.clientEp-> post(path, request, targetType = http:Response);
+        return response;
+    }
+    # Info for a specific pet
+    #
+    # + petId - The id of the pet to retrieve
+    # + 'limit - The id of the pet to retrieve
+    # # Deprecated parameters
+    # + 'limit - Limit is deprecated from v2
+    # + return - Expected response to a valid request
+    #
+    # # Deprecated
+    @deprecated
+  remote isolated function showPetById(string petId, @display {label: "Result limit"} @deprecated string 'limit) returns Pets|error {
+                                                                        string  path = string `/pets/${petId}`;
+                                                                        map<anydata> queryParam = {"limit": 'limit};
+        path = path + check getPathForQueryParam(queryParam);
+        Pets response = check self.clientEp-> get(path, targetType = Pets);
+        return response;
+    }
+}
+
+# Generate query path with query parameter.
+#
+# + queryParam - Query parameter map
+# + return - Returns generated Path or error at failure of client initialization
+isolated function  getPathForQueryParam(map<anydata> queryParam)  returns  string|error {
+    string[] param = [];
+    param[param.length()] = "?";
+    foreach  var [key, value] in  queryParam.entries() {
+        if  value  is  () {
+            _ = queryParam.remove(key);
+        } else {
+            if  string:startsWith( key, "'") {
+                 param[param.length()] = string:substring(key, 1, key.length());
+            } else {
+                param[param.length()] = key;
+            }
+            param[param.length()] = "=";
+                                                                                                              if  value  is  string {
+                                                                                                              string updateV =  check url:encode(value, "UTF-8");
+                                                                                                              param[param.length()] = updateV;
+                                                                                                              } else {
+                                                                                                              param[param.length()] = value.toString();
+                                                                                                              }
+                                                                                                              param[param.length()] = "&";
+  }
```

Comment:
Formatting issue.
Is it done on purpose or a mistake?

Reply:
Its a mistake. Thank you